doc/introduction/remote.rst

```diff
@@ -228,104 +154,122 @@ array([[0, 0, 1, 0, 1, 0, 1, 0],
+checked:
 
-    name compilation_example  # Name of the program
-    version 1.0               # Blackbird version number
-    target chip2 (shots=100)   # This program will run on chip0 for 100 shots
+>>> job.refresh()
+>>> job.status
+"complete"
+>>> job.result
+[[0 1 0 2 1 0 0 0]]
 
-    # Define a unitary matrix
-    complex array U[4, 4] =
-         0.09980516-0.78971535j,  0.53374613+0.07984545j, -0.21161788+0.10047649j, -0.01337026-0.14167555j
-         -0.12759979-0.00425289j,  0.14089156+0.40091225j, 0.31942372-0.21453252j, -0.79775306+0.13657774j
-         -0.18224807+0.30281836j,  0.26930442-0.04644871j, -0.46045639-0.55359506j, -0.0737605-0.52580999j
-         0.19903677-0.43076659j, -0.50320649-0.44750373j, -0.01617065-0.52755812j, -0.19729219+0.06200712j
+Finally, an incomplete job can be *cancelled* by calling :meth:`job.cancel() <~.Job.cancel>`.
 
-    # Initial states are two-mode squeezed states
-    S2gate(1.0, 0.0) | [0, 4]
-    S2gate(1.0, 0.0) | [1, 5]
-    S2gate(1.0, 0.0) | [2, 6]
-    S2gate(1.0, 0.0) | [3, 7]
 
-    # Apply the unitary matrix above to
-    # the first pair of modes, as well
-    # as a beamsplitter
-    Interferometer(U) | [0, 1, 2, 3]
-    BSgate(0.543, -0.123) | [0, 1]
+Hardware compilation
+--------------------
 
-    # Duplicate the above unitary for
-    # the second pair of modes
-    Interferometer(U) | [4, 5, 6, 7]
-    BSgate(0.543, -0.123) | [4, 5]
+When creating a quantum program to run on hardware, Strawberry Fields can compile
+any collection of the following gates into a multi-mode unitary:
 
-    # Perform a PNR measurement in the Fock basis
-    MeasureFock() | [0, 1, 2, 3, 4, 5, 6, 7]
+* `General beamsplitters <https://strawberryfields.readthedocs.io/en/stable/code/api/strawberryfields.ops.BSgate.html>`_ (:class:`~.ops.BSgate`),
 
+* `Mach-Zehnder interferometers <https://strawberryfields.readthedocs.io/en/stable/code/api/strawberryfields.ops.MZgate.html>`_ (:class:`~.ops.MZgate`), or
 
-**Note:** You may use :func:`~.random_interferometer` to generate arbitrary random unitaries.
+* `rotations/phase shifts <https://strawberryfields.readthedocs.io/en/stable/code/api/strawberryfields.ops.Rgate.html>`_ (:class:`~.ops.Rgate`).
 
-This program will execute following the same steps as above; :class:`~.RemoteEngine` will automatically
-compile the program to match the layout of the chip.
+Furthermore, several automatic decompositions are supported:
 
-You may wish to view the compiled program; this can be easily done in Python using
-the ``Program.print`` method:
+* You can use the :class:`~.ops.Interferometer` command to directly pass a
+  unitary matrix to be decomposed and compiled to match the device architecture.
+  This performs a rectangular decomposition using Mach-Zehnder interferometers.
 
+* You can use :class:`~.ops.BipartiteGraphEmbed` to embed a bipartite graph on
+  the GBS chip. Note, however, that the decomposed squeezing values depends on the graph
```

Comment:
better to actually make a callout Note here

Reply:
Made a warning box 🙂 
![image](https://user-images.githubusercontent.com/2959003/77224914-24942880-6bba-11ea-973c-d99f71d98375.png)